source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml

```
@@ -169,29 +178,25 @@ content: |
      .. tab::
         :tabid: java-sync
 
-        .. code-block:: Java
+        In your call to ``createDataKey()`` on a ``ClientEncryption`` keyvault
+        instance in ``CSFLEHelpers.java``, add your KMS provider and master 
+        key details as follows:
```

Small suggestion to make it less wordy.
```suggestion
        In your ``ClientEncryption.createDataKey()`` call in ``CSFLEHelpers.java``, add your KMS provider 
        and master key details as follows:
```

---

To me, this implies that `createDataKey()` is a static method, but it's an instance method. Maybe if I switch around the order of the sentence to:
"In `CSFLEHelpers.java`, add your KMS provider and master key details to your call to `createDataKey()` on your `ClientEncryption` instance."

Which loses some specificity into what ClientEncryption represents, but maybe that info is not that important.